doc/developing-the-website.md

```diff
@@ -14,4 +14,8 @@ Alternatively, to dynamically rebuild the site and refresh the browser _when cha
 
 ### Next steps
 
-When you are done with developing you can [build](building-the-website.md) and [deploy](deploying-the-website.md).
\ No newline at end of file
+When you are done with developing you can [build](building-the-website.md) and [deploy](deploying-the-website.md) manually.
+
+OR
```

Comment:
Is the manual deploy still an option at all? If Travis builds on every commit, you can't really deploy manually any more, right?

Reply:
You can! Incase travis builds break you may want to build + deploy manually.

A few usecases that this travis auto build + deploy doesn't cover

1) Doesn't auto pull in translation changes
2) Doesn't update latest (7.x) from dev version of docs. Need to do this manually
3) Doesn't create a new version of the docs (future 8.x).

Of course, you could do all 3 as commits and commit them and then travis will do the deploy.

Reply:
Got it.

1,2,3 should definitely be added to the autmated deploy documentation then.